doc/source/contributing/development_plan.rst

```diff
@@ -1,45 +1,51 @@
+Major Themes for Development
+----------------------------
 
-Improving Accessibility to Advanced Computing
----------------------------------------------
-Even as new resources and tools are developed to make computing easier, disciplinary researchers still face challenges finding time to maintain their computing environments while also meeting their research, teaching, and service objectives. Through integration of icepyx into the Pangeo ecosystem, only a handful of developers are able to manage the computational resources needed for working with ICESat-2 data, freeing researcher time and energy for exploring data. Further, the integration of Pangeo into existing advanced computing infrastructure (such as NASA's ADAPT) will enable researchers to take advantage of cloud computing without a significant need for re-tooling.
+Enhancing User Interactivity and Visualization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+icepyx aims to continually reduce the need for researchers to rewrite "routine" code by enabling easy visualization
+throughout the data inquiry to analyzed data presentation process and provide a simple, community-based framework for reproducibility.
 
 Open Science Example Use Cases
-------------------------------
-We are currently partnering with multiple researchers conducting investigations using ICESat-2 datasets. Their research, from data collection and analysis to publication, will be used to drive the development of icepyx functionality, ultimately providing software contributions and example workflows. Current collaborations focus on:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Research is the primary driver for development of icepyx functionality.
+If you are or plan to work on a project using ICESat-2 data, we encourage you to use icepyx as a framework for finding and processing your data, 
+from designing your analysis to writing code to analyze your data to generating presentation-quality figures.
+We welcome example use cases from all disciplines.
 
-- impacts of blowing snow and low clouds on ICESat-2 measurements
-- snow height in non-glaciated regions
-- parameter assimilation into sea ice models
```

Comment:
To better resonate with people (especially non-ice ones), it might be good to have a few examples of disciplines. E.g. glaciology, coastal geomorphology, vegetation biomass, etc. Maybe a quick search on what ICESat-2 has been used for will help out with this list.

Reply:
Agreed and think adding 'vegetation canopy height,' 'subsurface ocean structure,' 'sea level change' as suggestions

Reply:
So you would suggest NOT removing this? This original list was actually based on a non-funded proposal where specific groups had agreed to contribute examples as collaborators. Currently our examples center on "how to use icepyx" rather than research use cases. This would be good motivation for finishing out the pangeo-gallery steps to have a space for use case examples.

Reply:
> So you would suggest NOT removing this? This original list was actually based on a non-funded proposal where specific groups had agreed to contribute examples as collaborators.

Yes, don't remove the list, but do modify it to include newer stuff that people have used ICESat-2 for. My guess it that people want to see something that *clicks* with what they're doing.

> Currently our examples center on "how to use icepyx" rather than research use cases. This would be good motivation for finishing out the pangeo-gallery steps to have a space for use case examples.

Yeah, really need to reboot https://github.com/icesat2py/icepyx-gallery. I feel that there's been lots of improvements since 2020 to the whole Binder/Sphinx/Docker/NSIDC-auth ecosystem, and it's time to get that up and running again. Certainly could look at https://github.com/ICESAT-2HackWeek/website2022 and copy some of the techniques from there.

Reply:
Re-added a list of example topics here to show the disciplinary breadth.